component/notification/src/main/java/org/exoplatform/social/notification/plugin/SpaceInvitationPlugin.java

```diff
@@ -22,6 +22,8 @@
 import org.exoplatform.container.xml.InitParams;
 import org.exoplatform.social.core.space.model.Space;
 
+
+
```

Comment:
To remove, please always keep just the needed changes

Reply:
okay